tests/nsga2_test.cpp

```diff
@@ -412,4 +412,35 @@ TEST_CASE("NSGA2FonsecaFlemingTestVectorFloatBounds", "[NSGA2Test]")
   }
 
   REQUIRE(allInRange);
+}
+
+TEST_CASE("NSGA2ZDTONETest", "[NSGA2Test]")
```

Comment:
Can you add a simple description for the test on top?

Reply:
Also, do you mind to add a comment, that we also tested the other ZDT functions, but due to runtime constraints left didn't include them in the test suite?

Reply:
Done, do you mind confirming ?

Reply:
Thanks, looks perfect.